app/src/components/CustomPreset/CustomPreset.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import "./CustomPreset.css";
+import PresetIcon from "../../components/PresetIcon/PresetIcon";
+
+import { Container, TextField, Slider, Typography } from "@mui/material";
+
+function SliderComponent(props: any) {
+  return (
+    <Container>
+      <Typography>{props.label}</Typography>
+      <Slider step={0.1} min={0} max={1} marks valueLabelDisplay="auto" />
+    </Container>
+  );
+}
+
+function CustomPreset() {
+  return (
+    <div className="outerContainer">
+      <div className="customPresetContainer">
+        <PresetIcon />
+        <TextField
+          sx={{ width: "80%", borderRadius: "50px" }}
+          id="outlined-basic"
+          label="Preset Name..."
+          variant="filled"
+        />
+        <div className="sliderGridContainer">
+          <SliderComponent label={"Previously Used Ingredients"} />
+          <SliderComponent label={"Something New"} />
+          <SliderComponent label={"Short Cook Time"} />
+          <SliderComponent label={"Rating"} />
+          <SliderComponent label={"Few Ingredients"} />
+          <SliderComponent label={"Ingredients in Inventory"} />
+        </div>
+        <div className="buttonContainer">
+          <button className="button">Add Preset</button>
+          <button className="button">Cancel</button>
+        </div>
+      </div>
+    </div>
+  );
+}
+
+export default CustomPreset;
```

Comment:
I would prefer if the theme were more consistent with the other components (by wrapping everything inside a <Paper>)  so that it looks like this:

![image](https://github.com/JakobPaulsson/EasyEats/assets/57859068/9db8d39d-7f3d-4029-984a-00ccd3445362)

```suggestion
import React from "react";
import "./CustomPreset.css";
import PresetIcon from "../../components/PresetIcon/PresetIcon";

import { Container, TextField, Slider, Typography, Paper } from "@mui/material";

function SliderComponent(props: any) {
  return (
    <Container>
      <Typography>{props.label}</Typography>
      <Slider step={0.1} min={0} max={1} marks valueLabelDisplay="auto" />
    </Container>
  );
}

function CustomPreset() {
  return (
    <div className="outerContainer">
      <Paper
        elevation={6}
        sx={{
          width: 650,
          height: 800,
          mt: "60px",
          borderRadius: "10px",
        }}
      >
        <div className="customPresetContainer">
          <PresetIcon />
          <TextField
            sx={{ width: "80%", borderRadius: "50px" }}
            id="outlined-basic"
            label="Preset Name..."
            variant="filled"
          />
          <div className="sliderGridContainer">
            <SliderComponent label={"Previously Used Ingredients"} />
            <SliderComponent label={"Something New"} />
            <SliderComponent label={"Short Cook Time"} />
            <SliderComponent label={"Rating"} />
            <SliderComponent label={"Few Ingredients"} />
            <SliderComponent label={"Ingredients in Inventory"} />
          </div>
          <div className="buttonContainer">
            <button className="button">Add Preset</button>
            <button className="button">Cancel</button>
          </div>
        </div>
      </Paper>
    </div>
  );
}

export default CustomPreset;

```